### ISSUE TITLE
Import af DWG/DXF fra AutoCAD

### ISSUE THREAD

**Issue by HeidiZag** [qgisdk/forbedringer#12]
## Type af issue

*Vælg venligst typen af issue ved at sætte kryds i den relevante boks:*

- [ ] Fejlrettelse
- [ ] Forslag til ny funktion
- [ x] Generel udvikling

## Beskrivelse

**Detaljeret beskrivelse af problemet eller forslag:**

## Motivation og nytteværdi

*Forklar hvorfor denne funktion eller ændring er vigtig, og hvordan den vil gavne danske QGIS-brugere:*

## Links til eksisterende issues

*Hvis relevant, inkluder links til relaterede issues på den officielle QGIS [issue-side](https://github.com/qgis/QGIS/issues):*

## Skærmbilleder eller illustrationer

*Tilføj eventuelle skærmbilleder, diagrammer eller andre visuelle hjælpemidler:*

## Yderligere oplysninger

*Tilføj andre relevante oplysninger:*

**Comment by tinoks** [comment]
Har du nogle test data? Jeg plejer at have det modsatte problem, at hvis data er fra en udgave af Autocad tidligere end 2007 kan det ikke åbnes.

**Comment by HeidiZag** [comment]
Det er interessant og får mig til at tænke over om der er noget som planlæggerne ikke indlæser "korrekt". De importerer og laver en geopackage, som de så arbejder videre med. Der er næsten altid problemer med bygherres data vedr. koordinatsystem (eller mangel på samme) og enhed, men generelt er der også et versionsproblem - siger de. 
Når planlæggerne ikke kan få data ind i QGIS sender de filen til mig, hvor jeg "fixer" det med enhed og koordinatsystem i FME. Jeg prøver at få en planlægger til at sende en DXF/DWG som de mener ikke kan indlæses pga. versioneringsproblemer.

**Comment by tinoks** [comment]
Så prøver jeg at finde en gammel AutoCAD som ikke vil indlæse.

man. 16. dec. 2024 09.13 skrev Heidi Zagari ***@***.***>:

**Comment by HeidiZag** [comment]
Hej Tino
Jeg prøver at få en planlægger til at finde en fil, som ikke kan indlæses og som er en ny version.


Venlig hilsen

Heidi Zagari
GIS-koordinator

***@***.***

By- og Miljøcenter
BMC - Geodata
Rådhusstræde 1
2630 Taastrup
Telefon: 43591000
Direkte: 43591295
www.htk.dk
EANnr: 5790001123208
***@***.***

Fra: Tino Kastbjerg Stigsen ***@***.***>
Sendt: 13. december 2024 15:48
Til: qgisdk/forbedringer ***@***.***>
Cc: Heidi Zagari ***@***.***>; Author ***@***.***>
Emne: Re: [qgisdk/forbedringer] Import af DWG/DXF fra AutoCAD (Issue #12)


Har du nogle test data? Jeg plejer at have det modsatte problem, at hvis data er fra en udgave af Autocad tidligere end 2007 kan det ikke åbnes.

**Comment by MetteHJ** [comment]
Hej
Faldt over jeres tråd i jagten på løsning af lign. problem.
"Import af tegning mislykkedes (unsupported version. Cannot read AutoCAD 2018/2019/2020 documents.)"

[Helsingorsvej_39.zip](https://github.com/user-attachments/files/18199498/Helsingorsvej_39.zip)

**Comment by HeidiZag** [comment]
Det er godt du bidrager her, så vi kan finde ud om der er flere der oplever denne udfordring.

**Comment by MetteHJ** [comment]
Jeg fandt en løsning i dette lille stykke software, gratis prøveversion transformerede alt uden brok og jeg er videre i min opgave:
[https://www.guthcad.com/cad2shape.htm]
Har ønsket mig en licens i julegave af chefen!

**Comment by HeidiZag** [comment]
Fedt, du fandt guthcad - og det virker. Tak for at sende link..
Jeg vil selvfølgelig gerne at QGIS fællesskabet kan finde nogen der kan lave en opgradering af det eksisterende python.

**Comment by tinoks** [comment]
En løsning kan være at få noget i QGIS som konvertere filerne tilbage til AutoCAD Version 2013. 
Jeg har denne fil som ikke virker før den er bagkonverteret.
[LM 678 Grundlag.zip](https://github.com/user-attachments/files/18344918/LM.678.Grundlag.zip)

**Comment by tinoks** [comment]
> En løsning kan være at få noget i QGIS som konvertere filerne tilbage til AutoCAD Version 2013. Jeg har denne fil som ikke virker før den er bagkonverteret. [LM 678 Grundlag.zip](https://github.com/user-attachments/files/18344918/LM.678.Grundlag.zip)
Skal se sådan her ud:
![image](https://github.com/user-attachments/assets/0c9ed69d-c2f2-4474-90d5-10f00cf42087)

**Comment by HeidiZag** [comment]
Jo, men umiddelbart lidt bagvendt. Det vil dog i første omgang kunne hjælpe brugerne til at komme videre uden at skulle kontakte GIS medarbejderne med FME (eller lignende programmer) for at konvertere.

**Comment by HeidiZag** [comment]
Hej igen (Tino). Nu fik jeg endelig en en DWG fil fra en planlægger som han ikke kan læse ind. 


[Selsmosen Situationsplan_S34 sjælland_meter_KJark_UTMzone32euref89.zip](https://github.com/user-attachments/files/18521223/Selsmosen.Situationsplan_S34.sjaelland_meter_KJark_UTMzone32euref89.zip)

**Comment by tinoks** [comment]
I 3.44 får jeg dette:

![Image](https://github.com/user-attachments/assets/22e148c3-2de0-4936-b296-49fc2e14bd29)

Er det korrekt?
